.cargo/config

```diff
@@ -1,2 +1,5 @@
 [alias]
-xtask = "run --package xtask --"
\ No newline at end of file
+xtask = "run --package xtask --"
+
+[target.x86_64-pc-windows-gnu]
+linker = "x86_64-w64-mingw32-gcc"
```

Comment:
can you enlighten me?

Reply:
This is so you can cross compile for windows on mac and I don't have to go looking for a windows box.